spec/rollbar/item_spec.rb

```diff
@@ -291,7 +297,7 @@
       context 'with no exception' do
         let(:exception) { nil }
 
-        it 'should build a message body when no exception is passed in' do
+        it 'should build a message body' do
```

Comment:
This, and other examples like this one, already have these when/with modifiers in the enclosing context label. (At line 297 in this case.)